graylog/package.yaml

```diff
@@ -0,0 +1,81 @@
+---
+id: graylog
+name: Graylog
+author: Tenzir
+author_icon: https://raw.githubusercontent.com/tenzir/library/main/author.svg
```

Comment:
Maybe use [this logo](https://www.svgrepo.com/svg/353838/graylog-icon) as `package_icon`.

Reply:
I'll trust that the license is suitable.